nautobot/dcim/factory.py

```diff
@@ -644,6 +642,9 @@ class Params:
     )
     image_file_size = factory.Maybe("has_image_file_size", factory.Faker("pyint"), None)
     download_url = factory.Maybe("has_download_url", factory.Faker("uri"), "")
+    default_image = factory.LazyAttribute(
+        lambda o: not o.software_version.software_image_files.filter(default_image=True).exists()
+    )
```

Comment:
This makes it so the first image file generated for each version gets tagged as default in all cases, right? Is that acceptable/desirable?

Reply:
It's probably not the most accurate but I don't think it would be trivial to get it perfect and I'm manually updating the field in the tests where it matters.